README.md

```diff
@@ -51,7 +26,7 @@ module.exports = {
 
 ```javascript
 "scripts": {
-    "lint": "eslint app/**/*.{js,jsx} __tests__/**/*.{js,jsx}",
+    "lint": "eslint --color src/**/*.{js,jsx,ts,tsx}",
```

Comment:
nit: dont ❤️ specifying any subfolder here, but it does suck to have it bare and default to scanning the `node_js` folder.  so... idc, probably ok assuming you used create-react-app

Reply:
I think that's a good call out. It's probably worth explaining this example a bit and offering alternatives: 80af6adb2bf2839d617e093fc7d95377574345b8